docs/libraries/json-schema/reference/decorators.md

```diff
@@ -272,9 +272,9 @@ Specify that the target array must begin with the provided types.
 
 #### Parameters
 
-| Name  | Type              | Description                                                                 |
-| ----- | ----------------- | --------------------------------------------------------------------------- |
-| value | `model unknown[]` | a tuple containing the types that must be present at the start of the array |
+| Name  | Type    | Description                                                                 |
+| ----- | ------- | --------------------------------------------------------------------------- |
+| value | `Array` | a tuple containing the types that must be present at the start of the array |
```

Comment:
It says this is a tuple but the type is array?

Reply:
well the type is an array and you pass a tuple which is an fixed length array constant.